src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageSearchMetadataContextInfo.cs

```diff
@@ -74,9 +75,10 @@ public static PackageSearchMetadataContextInfo Create(IPackageSearchMetadata pac
                 KnownOwners = knownOwners,
                 ReportAbuseUrl = packageSearchMetadata.ReportAbuseUrl,
                 PackageDetailsUrl = packageSearchMetadata.PackageDetailsUrl,
+                RawReadmeUrl = packageSearchMetadata.RawReadmeUrl,
                 PackagePath =
                     (packageSearchMetadata as LocalPackageSearchMetadata)?.PackagePath ??
-                    (packageSearchMetadata as ClonedPackageSearchMetadata)?.PackagePath,
+                    (packageSearchMetadata as PackageSearchMetadataBuilder.ClonedPackageSearchMetadata)?.PackagePath,
```

Comment:
Why was this necessary here?

Reply:
I swear it was giving me an error earlier, removed 🤷